invenio_app_ils/documents/schemas/documents/document-v1.0.0.json

```diff
@@ -728,6 +728,10 @@
       "minItems": 1,
       "type": "array",
       "uniqueItems": true
+    },
+    "physical_description": {
```

Comment:
:( this should ideally go to the Item and not here, but if this change is needed...
We will have also to add this to the React-JSONSchema form schema.

Reply:
the problem is that if we have `physical_description` it will be copied over each instance of item but in the same time it is more flexible if one of the items is different than the others. WDYT?